File: CaseStudy/stock/src/main/java/com/stores/stock/repository/BilledProductsRepository.java
package com.stores.stock.repository;

import com.stores.stock.model.BilledProduct;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface BilledProductsRepository extends JpaRepository<BilledProduct, Integer> {
}

File: CaseStudy/demo_kotlin/src/main/kotlin/com/store/demo/Repository/BilledProductsRepository.kt
package com.store.demo.Repository

import com.store.demo.model.BilledProduct
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
@Repository
interface BilledProductsRepository : JpaRepository<BilledProduct?, Int?> {
    fun findByBillId(id: Int): List<BilledProduct?>?
}